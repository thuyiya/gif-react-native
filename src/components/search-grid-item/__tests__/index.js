/* eslint-disable no-undef */
import React from "react";
// import TestRenderer from "react-test-renderer";
import ShallowRenderer from "react-test-renderer/shallow";
import { SearchGridItem } from "_components";

describe("SearchGridItem", () => {
  it("renders correctly react-test-renderer", () => {
    const renderer = new ShallowRenderer();
    renderer.render(
      <SearchGridItem onPress={() => {}} source={""} title={""} />
    );
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  });
});
