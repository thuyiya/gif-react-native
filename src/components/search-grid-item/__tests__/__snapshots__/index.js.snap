// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchGridItem renders correctly react-test-renderer 1`] = `
<TouchableOpacity
  accessibilityHint="Tap to navigate to full search view"
  accessibilityLabel="Search Item of "
  accessible={true}
  onPress={[Function]}
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "padding": 4,
    }
  }
>
  <Image
    resizeMode="cover"
    source={
      Object {
        "uri": "",
      }
    }
    style={
      Object {
        "height": 231.33333333333334,
        "width": 231.33333333333334,
      }
    }
  />
</TouchableOpacity>
`;
